config/jobs/kubernetes/sig-cloud-provider/gcp/gcp-gce.yaml

```diff
@@ -814,13 +814,12 @@ periodics:
       - --extract=ci/fast/latest-fast
       - --extract-ci-bucket=k8s-release-dev
       - --gcp-master-image=gci
-      - --gcp-node-image=gci
-      - --gcp-nodes=4
```

Comment:
If there's only 1 node, would that be enough to run the same test suite in parallel? I remember that storage tests rely on detaching volumes from a node a reattaching them to a new node, how would that work here?

Reply:
The release branch jobs seem to run on single node as they don't pas the `--gcp-nodes` flag.
As those jobs are running the same test_args, I thought one node would suffice.

@mauriciopoppe Can you please list down the tests that you feel might need multi node environment.
https://testgrid.k8s.io/sig-release-1.32-blocking#gce-cos-k8sbeta-default is the equivalent job for `1.32` release branch.
https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes/generated/generated.yaml#L61:L72

Reply:
Thanks for the testgrid link, I couldn't find multivolume tests from storage. I'm also not sure that if you don't pass `--gcp-nodes` that it'll default to 1 node, here it says it defaults to 3 https://github.com/kubernetes-sigs/kubetest2/blob/22d5b1410bef09ae679fa5813a5f0d196b6079de/kubetest2-gce/deployer/common.go#L143